stablehlo/tests/interpret_real.mlir

```diff
@@ -0,0 +1,32 @@
+// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
+
+// CHECK-LABEL: Evaluated results of function: real_op_test_f64
+func.func @real_op_test_f64() -> tensor<11xf64> {
+  %0 = stablehlo.constant dense<[0.0, -0.0, 1.0, 0.125, 0.1, 3.1415926535897931, 0x7FF0000000000000, 0xFFF0000000000000, 0x7FFFFFFFFFFFFFFF, 0x0000000000000001, 0x8000000000000001]> : tensor<11xf64>
```

Comment:
This test can be shortened to include a few normal values. No need to test with Nan, Infs or denormals.